### PR TITLE
[USM] Added traced programs to the USM status

### PR DESCRIPTION
**Pull request by amitslavin**
### What does this PR do?

Adding USM traced programs to the USM status will be reflected in the flare file as well.

### Motivation

Improve USM debugging tools.

### Additional Notes

The flare/status check after the change:

```
  universal_service_monitoring:
    last_check: 1.717513387e+09
    state: running
    traced_programs:
    - FilePath: /usr/lib/aarch64-linux-gnu/libcrypto.so.3
      PIDs:
      - 732
      - 1648
      - 1711
      - 37852
      - 38905
      - 39486
      - 451
      - 38291
      - 38909
      - 40059
      - 40703
      - 716
      - 1652
      - 1715
      - 37856
      - 38740
      - 39482
      - 39944
      - 494
      - 1660
      - 774
      - 734
      - 792
      - 1714
      - 38295
      - 39344
      - 40402
      - 1
      - 38744
      - 39940
      - 40707
      - 40796
      - 40890
      - 1651
      - 39340
      - 40055
      - 779
      - 40406
      ProgramType: shared_libraries
    - FilePath: /usr/lib/aarch64-linux-gnu/libgnutls.so.30.31.0
      PIDs:
      - 734
      ProgramType: shared_libraries
    - FilePath: /usr/bin/dockerd
      PIDs:
      - 1008
      ProgramType: go-tls
```
Example whiteout any traced programs:

```
network_tracer:
  tracer:
    last_check: 1.717569736e+09
  universal_service_monitoring:
    state: disabled
    traced_programs: null
```

### CODE REVIEW

File: pkg/network/usm/monitor.go [hunk 1]
```diff
@@ -156,6 +157,11 @@ func (m *Monitor) GetUSMStats() map[string]interface{} {
 		response["error"] = startupError.Error()
 	}
 
+	tracedPrograms := utils.GetTracedProgramList()
+	if len(tracedPrograms) > 0 {
+		response["traced_programs"] = tracedPrograms
+	}
```
[the remark this answers is not in the document]
Comment (amitslavin) [reply pkg/network/usm/monitor.go]: done

[the remark this answers is not in the document]
Comment (guyarb) [reply pkg/network/usm/monitor.go]: can you add an example for no traced program?

[the remark this answers is not in the document]
Comment (amitslavin) [reply pkg/network/usm/monitor.go]: done

**Review by guyarb — APPROVED** [review]

**Pull request merged**